x-pack/plugin/core/src/main/java/org/elasticsearch/xpack/core/security/authz/store/KibanaOwnedReservedRoleDescriptors.java

```diff
@@ -522,6 +522,7 @@ static RoleDescriptor kibanaSystem(String name) {
                         "logs-google_scc.finding-*",
                         "logs-aws.securityhub_findings-*",
                         "logs-aws.securityhub_findings_full_posture-*",
+                        "logs-aws_securityhub.finding-*",
```

Comment:
Just out of curiosity: what is the difference in the data stored in `"logs-aws.securityhub_findings-*"` that we already have above and ` "logs-aws_securityhub.finding-*"`? Is the former for the list of findings and the latter for the details of a specific finding or ...?